contracts/products/SoteriaCoverageProduct.sol

```diff
@@ -190,122 +199,68 @@ contract SoteriaCoverageProduct is ISoteriaCoverageProduct, ERC721, EIP712, Reen
-     */
-     function cancelPolicy(uint256 policyID_) external override nonReentrant {
-        require(_exists(policyID_), "invalid policy");
-        require(ownerOfPolicy(policyID_) == msg.sender, "!policyholder");
-        uint256 refundAmount = funds(msg.sender);
-        _activeCoverAmount -= coverAmountOf(policyID_);
-        _coverAmountOf[policyID_] = 0;
-
-        // send deposited fund to the policyholder
-        if (refundAmount > 0) Address.sendValue(payable(msg.sender), refundAmount);
-        _funds[msg.sender] = 0;
-        _updatePolicyManager(_activeCoverAmount);
-        emit PolicyCanceled(policyID_);
-    }
-
-    /**
-     * @notice Submit a claim.
-     * The user can only submit one claim per policy and the claim must be signed by an authorized signer.
-     * If successful the policy is not burnt and a new claim is created.
-     * The new claim will be in [`ClaimsEscrow`](../ClaimsEscrow) and have the same ID as the policy.
-     * Can only be called by the policyholder.
-     * @param policyID_ The policy that suffered a loss.
-     * @param amountOut_ The amount the user will receive.
-     * @param deadline_ Transaction must execute before this timestamp.
-     * @param signature_ Signature from the signer.
+     * @notice Deactivate a user's own policy.
+     * @param policyID_ The policy ID to update.
+     * User will receive their entire Soteria account balance.
      */
-     function submitClaim(
-        uint256 policyID_,
-        uint256 amountOut_,
-        uint256 deadline_,
-        bytes calldata signature_
-    ) external override nonReentrant {
-        // validate inputs
-        require(policyStatus(policyID_), "inactive policy");
-        // solhint-disable-next-line not-rely-on-time
-        require(block.timestamp <= deadline_, "expired deadline");
-        require(ownerOfPolicy(policyID_) == msg.sender, "!policyholder");
-        require(amountOut_ <= coverAmountOf(policyID_), "excessive amount out");
-        // verify signature
-        {
-        bytes32 structHash = keccak256(abi.encode(_SUBMIT_CLAIM_TYPEHASH, policyID_, msg.sender, amountOut_, deadline_));
-        bytes32 hash = _hashTypedDataV4(structHash);
-        address signer = ECDSA.recover(hash, signature_);
-        require(_isAuthorizedSigner[signer], "invalid signature");
-        }
-        _activeCoverAmount -= coverAmountOf(policyID_);
-        _coverAmountOf[policyID_] = 0;
-        _updatePolicyManager(_activeCoverAmount);
-        // submit claim to ClaimsEscrow
-        IClaimsEscrow(payable(_registry.claimsEscrow())).receiveClaim(policyID_, msg.sender, amountOut_);
-        emit ClaimSubmitted(policyID_);
+     function deactivatePolicy(uint256 policyID_) public override nonReentrant {
+        require(policyStatus(policyID_), "invalid policy");
+        address policyOwner = ownerOf(policyID_);
+        require(this.governance() == msg.sender || policyOwner == msg.sender, "Not owner or governance");
```

Comment:
Lowercase the message. **"not owner or governance**"

Reply:
Done